lib/app/modules/quality_reporting/valueset_sequence.rb

```diff
@@ -21,13 +20,8 @@ class ValueSetSequence < SequenceBase
           desc 'Expand each Value Set in a measure to ensure they are available'
         end
 
-        root = "#{__dir__}/../../../.."
-        path = File.expand_path('resources/quality_reporting/CMS130/Bundle/cms130-bundle.json', root)
-        bundle = FHIR::STU3::Bundle.new JSON.parse(File.read(path))
-        measure_id = 'MitreTestScript-measure-col'
-        measure = get_resource_by_id(bundle, measure_id)
-        valueset_urls = get_all_dependent_valuesets(measure, bundle)
-
+        measure_id = @instance.measure_to_test
```

Comment:
We should assert that `measure_id` is not `nil` here so the sequence doesn't pass if there isn't a measure tested.

Reply:
Added this check. Can test using an empty cqf-ruler instance.